frameworks/template/cli/dcos-template/.gitignore

```diff
@@ -1,2 +1,3 @@
 dcos-*
 *.whl
+native-*
```

Comment:
I think every go build directory would need this, not just `helloworld` and `template` CLIs. These are generated by tools/build_go_exe.sh.

- `frameworks/*/cli/*/.gitignore` 
- `sdk/bootstrap/.gitignore`